apps/crc_sus.py

```diff
@@ -38,20 +34,12 @@ def get_allocation_info(self, account: str) -> Dict[str, int]:
             A dictionary mapping cluster names to the number of service units
         """
 
-        # Connect to the database and get the table with proposal service units
-        database = dataset.connect(self.banking_db_path, sqlite_wal_mode=False)
-        table = database['proposal']
-
-        # Ensure a proposal exists for the given account
-        db_record = table.find_one(account=account)
-        if db_record is None:
-            raise ValueError('ERROR: No proposal for the given account was found')
+        acct = AccountServices(account)
+        allocs = acct._get_active_proposal_allocation_info()
```

Comment:
@djperrefort I've abstracted away the sqlalchemy here and in crc-sus. Is this how you envisioned it utilizing code internal to the bank?

Reply:
This is it exactly. In principle we should provide dedicated methods for data access in the bank package, but we can leave that for the future.